docs/docs/0.4.0/policies/fault-injection.md

```diff
@@ -0,0 +1,95 @@
+# Fault Injection
+
+`FaultInjection` policy helps you to test your microservices against resiliency. Kuma provides 3 different types of failures that could be imitated in your environment. 
+These faults are [Delay](#delay), [Abort](#abort) and [ResponseBandwidth](#responsebandwidth-limit) limit.
+
+::: warning
+Please note, `FaultInjection` operates on L7 HTTP traffic, so make sure that selected dataplanes are configured with HTTP Protocol.
+:::
+On Universal:
+
+```yaml
+type: FaultInjection
+mesh: default
+name: fi1
+sources:
+    - match:
+        service: frontend
+        version: "0.1"
+destinations:
+    - match:
+        service: backend
+conf:        
+    abort:
+        httpStatus: 500
+        percentage: 50
+    delay:
+        percentage: 50
+        value: 5s
+    responseBandwidth:
+        limit: 50 mbps
+        percentage: 50    
+```
+
+On Kubernetes:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: FaultInjection
+mesh: default
+metadata:
+  namespace: default
+  name: fi1
+spec:
+    sources:
+        - match:
+            service: frontend
+            version: "0.1"
+            protocol: http
+    destinations:
+        - match:
+            service: backend
+            protocol: http
+    conf:        
+        abort:
+            httpStatus: 500
+            percentage: 50
+        delay:
+            percentage: 50
+            value: 5s
+        responseBandwidth:
+            limit: 50 mbps
+            percentage: 50 
+```
+
+### Sources & Destinations
+`FaultInjection` is a Connection policy, which means it is applied to the connection between dataplanes. As most of the policies, `FaultInjection` supports the powerful mechanism of matching, which allows you to precisely match source and destination services.
+
+::: warning
+Since `FaultInjection` policy available only for L7 HTTP traffic, `protocol: http` is mandatory tag both for source and destination selector.
```

Comment:
The warning at the top tells me the same message. Can we merge those and have only this one?